Lib/test/libregrtest/runtest.py

```diff
@@ -143,6 +143,14 @@ def __str__(self) -> str:
 # set of tests that we don't want to be executed when using regrtest
 NOTTESTS = set()
 
+# If these test directories are encountered recurse into them and treat each
+# test_ .py or dir as a separate test module. This can increase parallelism.
+# Beware this can't generally be done for any directory with sub-tests as the
+# __init__.py may do things which alter what tests are to be run.
+SPLITTESTDIRS = {
+    "test_asyncio",
+    "test_compiler",
```

Comment:
`test_compiler` is actually cinder-only, so this PR can be just for `test_asyncio`; sorry I should have mentioned that before. Having this here is a no-op since `test_compiler` will just never be encountered, but it'll be confusing to readers :)

Reply:
Okay removed test_compiler.